server/database/models/users.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Users = sequelize.define('Users', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    avatarUrl: DataTypes.STRING,
+    verified: DataTypes.BOOLEAN
+  }, {});
+  Users.associate = function(models) {
```

Comment:
I think you should remove these associations since the other models linked with the user model have not been created yet.

Reply:
okay, I agree with you and when other models are added, they should add it then

Reply:
Yes